### PR TITLE
Final adjustments before major version release

### PR DESCRIPTION
**Pull request by Blackbaud-SteveBrush**

### CODE REVIEW

**Comment by Blackbaud-BobbyEarl** [comment]
Preview build at [https://host.nxt.blackbaud.com/skyux-docs-tools?_sp.spa=pr-final-adjustments](https://host.nxt.blackbaud.com/skyux-docs-tools?_sp.spa=pr-final-adjustments)

File: src/app/public/modules/demo-page/demo-page-dom-adapter.service.ts [hunk 1]
```diff
@@ -0,0 +1,93 @@
...
+
+import {
+  SkyDocsAnchorLink
+} from '../type-definitions/anchor-link';
+
+/**
+ * @internal
+ */
+@Injectable()
+export class SkyDocsDemoPageDomAdapterService implements OnDestroy {
```
Comment (Blackbaud-SteveBrush) [inline src/app/public/modules/demo-page/demo-page-dom-adapter.service.ts]: No new logic, here. Just moving the DOM-related code from the demo-page component to a separate adapter service.

File: src/app/public/modules/type-definitions/pipe-definition.component.html [hunk 2]
```diff
@@ -28,7 +28,7 @@ <h4 class="sky-docs-pipe-definition-heading">
 
   <sky-docs-parameter-definitions>
     <sky-docs-parameter-definition
-      [parameterName]="config.inputValue.type"
+      [parameterType]="config.inputValue.type"
```
Comment (Blackbaud-SteveBrush) [inline src/app/public/modules/type-definitions/pipe-definition.component.html]: Wrong input was used; this fixes it.

File: src/app/public/styles/docs-tools.scss [hunk 3]
```diff
@@ -48,4 +48,9 @@
       margin-top: 0 !important;
     }
   }
-}
\ No newline at end of file
+}
+
+.sky-docs-text-block {
```
Comment (Blackbaud-SteveBrush) [inline src/app/public/styles/docs-tools.scss]: This global CSS class is a migration of the `skyux2-docs` class, `.sky-text-block` per Adam's request.

**Pull request merged**